birdspotter/requirements.txt

```
@@ -5,7 +5,9 @@ click==7.1.2
 click-plugins==1.1.1
 cligj==0.7.0
 Django==3.1.3
+django-widget-tweaks==1.4.8
 Fiona==1.8.17
+GDAL==3.2.0
```

Is this added as an explicit requirement (as opposed to a implicit recursive dependency) for a reason?
regardless, the main branch is currently running on GDAL 3.2.1
```suggestion
GDAL==3.2.1
```